conf/modules/rank_variants.config

```diff
@@ -16,34 +16,56 @@
 //
 
 process {
-    withName: '.*RANK_VARIANTS_SV:.*' {
-        ext.when = !params.skip_sv_annotation
-        publishDir = [
-            enabled: false
-        ]
-    }
+    if (!params.skip_sv_annotation) {
+        withName: '.*RANK_VARIANTS_SV:.*' {
+            publishDir = [
+                enabled: false
+            ]
+        }
 
-    withName: '.*RANK_VARIANTS_SV:GENMOD_ANNOTATE' {
-        ext.args = {
-            (params.genome == 'GRCh37') ? '--annotate_regions --genome-build 37' : '--annotate_regions --genome-build 38'
+        withName: '.*RANK_VARIANTS_SV:GENMOD_ANNOTATE' {
+            ext.prefix = { "${meta.id}_sv_genmod_annotate_${meta.set}" }
+            ext.args = { [
+                '--annotate_regions',
+                params.genome.equals('GRCh37') ? '--genome-build 37' : '--genome-build 38'
```

Comment:
Is it safe to assume that when `params.genome` isn't equal to `GRCh37` then it is GRCh38?

Reply:
Good point! We can update the schema to only allow for those two? What do you think @ramprasadn?

Reply:
Yupe.. the default is set in nextflow.config. But thanks for bringing it up. Just realized we never restricted what values this parameter could take. Will fix it 😅